runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -6721,6 +6722,16 @@ static int32_t J9THREAD_PROC samplerThreadProc(void * entryarg)
                {
                CalculateOverallCompCPUUtilization(compInfo, crtTime, samplerThread);
                }
+#if defined(J9VM_OPT_JITSERVER) && defined(LINUX)
+         static uint64_t lastMallocTrimTime = 0;
```

Comment:
Doesn't this need to be indented more?
Since the next block (line 6736) already has `#if defined(J9VM_OPT_JITSERVER)` , maybe insert this new code under the same ifdef and just add `#if defined(LINUX)`